### ISSUE TITLE
Fails to open ppm and pgm files containing comments

### ISSUE THREAD

**Issue by nhatzHK** [derf/feh#430]
When I try to open a ppm or pgm file containing comments, like
```
#comments 
P2
256 192
255
97 91 98 94 91 ...
```
which is a totally fine file, I get the following output
```
feh WARNING: file.ppm - No Imlib2 loader for that file format
feh: No loadable images specified.
See 'feh --help' or 'man feh' for detailed usage information
```
If I remove the comments the file is displayed correctly though.

**Comment by Nopey** [comment]
You have broken the magic number `P2` (newline)
Magic numbers (the first few bytes of a file) are used by feh, and similar programs, to tell what type a file is without relying upon the file extension.
According to [the pgm format specification](http://netpbm.sourceforge.net/doc/pgm.html), the first three characters need to be P, the number (Depending on the filetype, one of 1?, 2, ... 7)
I was able to recreate both the error, and feh succeeding when the comment was moved below the magic number.

TLDR; Move the comments below the `P2`, this is a malformed input file, not a bug in feh

**Comment by nhatzHK** [comment]
You're right. Thanks for the heads up. This can be closed.

**Comment by tankorsmash** [comment]
I had a similar issue, in that my ppm file was using bytes for the size, but it needs to be characters. so `"200"` instead of `È`. Thanks for `feh`!
